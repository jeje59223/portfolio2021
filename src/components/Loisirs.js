import '../styles/loisirs.css';
// eslint-disable-next-line no-unused-vars
import { Carousel } from 'react-bootstrap';
import data from '../data/loisirs.json';

const Loisirs = () => (
        <section id="loisirs" className="py-2">
            <h2 className="text-center h2Loisirs">Mes Loisirs</h2>
            <div id="carouselContent">
            <Carousel>
            {
                data.map((loisir) => (
                        <Carousel.Item key={loisir.id}>
                            <div className="text-center p-4">
                                <p className="font-italic">{loisir.title}</p>
                            </div>
                            <Carousel.Caption>
                            </Carousel.Caption>
                        </Carousel.Item>
                ))
            }
            </Carousel>
            </div>
        </section>
);

export default Loisirs;
