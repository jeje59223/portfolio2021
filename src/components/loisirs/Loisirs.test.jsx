import toJson from 'enzyme-to-json';
import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Loisirs from './Loisirs';

configure({ adapter: new Adapter() });

describe('<Loisirs>', () => {
  it('should render Loisirs Component', () => {
    const wrapper = shallow(<Loisirs />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});
