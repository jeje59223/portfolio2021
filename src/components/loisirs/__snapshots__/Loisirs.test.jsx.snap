// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loisirs> should render Loisirs Component 1`] = `
<section
  className="py-2"
  id="loisirs"
>
  <h2
    className="text-center h2Loisirs"
  >
    Mes Loisirs
  </h2>
  <div
    id="carouselContent"
  >
    <Carousel
      controls={true}
      defaultActiveIndex={0}
      fade={false}
      indicators={true}
      interval={5000}
      keyboard={true}
      nextIcon={
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
      }
      nextLabel="Next"
      pause="hover"
      prevIcon={
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
      }
      prevLabel="Previous"
      slide={true}
      touch={true}
      wrap={true}
    >
      <CarouselItem
        key="1"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Programmation de robots via des cartes Arduino
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
      <CarouselItem
        key="2"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Le Football, le Cyclisme et le Ski
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
      <CarouselItem
        key="3"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Le Bricolage et le Jardinage
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
      <CarouselItem
        key="4"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Les Jeux Vidéo
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
      <CarouselItem
        key="5"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Passer du temps avec mes filles !
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
      <CarouselItem
        key="6"
      >
        <div
          className="text-center p-4"
        >
          <p
            className="font-italic"
          >
            Réaliser des projets perso dans le but d'apprendre de nouvelles techno
          </p>
        </div>
        <CarouselCaption />
      </CarouselItem>
    </Carousel>
  </div>
</section>
`;
